tools/cockpit.spec

```diff
@@ -402,7 +402,8 @@ authentication via sssd/FreeIPA.
 %{_unitdir}/cockpit-wsinstance-https@.socket
 %{_unitdir}/cockpit-wsinstance-https@.service
 %{_unitdir}/system-cockpithttps.slice
-%{_prefix}/%{__lib}/tmpfiles.d/cockpit-tempfiles.conf
+%{_prefix}/%{__lib}/tmpfiles.d/cockpit-tmpfiles.conf
+%{_sysusersdir}/cockpit-sysusers.conf
```

Comment:
Rebase error, this should be in the next commit, not the "rename tmpfiles" commit.